apps/lastfm/last_fm.star

```diff
@@ -0,0 +1,164 @@
dDqyOu87LTtrO6o76DwnPGU8pDzjPSI9YT2hPeA+ID5gPqA+4D8hP2E/oj/iQCNAZECmQOdBKUFqQaxB7kIwQnJCtUL3QzpDfUPARANER0SKRM5FEkVVRZpF3kYiRmdGq0bwRzVHe0fASAVIS0iRSNdJHUljSalJ8Eo3Sn1KxEsMS1NLmkviTCpMcky6TQJNSk2TTdxOJU5uTrdPAE9JT5NP3VAnUHFQu1EGUVBRm1HmUjFSfFLHUxNTX1OqU/ZUQlSPVNtVKFV1VcJWD1ZcVqlW91dEV5JX4FgvWH1Yy1kaWWlZuFoHWlZaplr1W0VblVvlXDVchlzWXSddeF3JXhpebF69Xw9fYV+zYAVgV2CqYPxhT2GiYfViSWKcYvBjQ2OXY+tkQGSUZOllPWWSZedmPWaSZuhnPWeTZ+loP2iWaOxpQ2maafFqSGqfavdrT2una/9sV2yvbQhtYG25bhJua27Ebx5veG/RcCtwhnDgcTpxlXHwcktypnMBc11zuHQUdHB0zHUodYV14XY+dpt2+HdWd7N4EXhueMx5KnmJeed6RnqlewR7Y3vCfCF8gXzhfUF9oX4BfmJ+wn8jf4R/5YBHgKiBCoFrgc2CMIKSgvSDV4O6hB2EgITjhUeFq4YOhnKG14c7h5+IBIhpiM6JM4mZif6KZIrKizCLlov8jGOMyo0xjZiN/45mjs6PNo+ekAaQbpDWkT+RqJIRknqS45NNk7aUIJSKlPSVX5XJljSWn5cKl3WX4JhMmLiZJJmQmfyaaJrVm0Kbr5wcnImc951kndKeQJ6unx2fi5/6oGmg2KFHobaiJqKWowajdqPmpFakx6U4pammGqaLpv2nbqfgqFKoxKk3qamqHKqPqwKrdavprFys0K1ErbiuLa6hrxavi7AAsHWw6rFgsdayS7LCszizrrQltJy1E7WKtgG2ebbwt2i34LhZuNG5SrnCuju6tbsuu6e8IbybvRW9j74KvoS+/796v/XAcMDswWfB48JfwtvDWMPUxFHEzsVLxcjGRsbDx0HHv8g9yLzJOsm5yjjKt8s2y7bMNcy1zTXNtc42zrbPN8+40DnQutE80b7SP9LB00TTxtRJ1MvVTtXR1lXW2Ndc1+DYZNjo2WzZ8dp22vvbgNwF3IrdEN2W3hzeot8p36/gNuC94UThzOJT4tvjY+Pr5HPk/OWE5g3mlucf56noMui86Ubp0Opb6uXrcOv77IbtEe2c7ijutO9A78zwWPDl8XLx//KM8xnzp/Q09ML1UPXe9m32+/eK+Bn4qPk4+cf6V/rn+3f8B/yY/Sn9uv5L/tz/bf///9sAQwAKBwcIBwYKCAgICwoKCw4YEA4NDQ4dFRYRGCMfJSQiHyIhJis3LyYpNCkhIjBBMTQ5Oz4+PiUuRElDPEg3PT47/9sAQwEKCwsODQ4cEBAcOygiKDs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7/8AAEQgAJgAmAwERAAIRAQMRAf/EAB8AAAEFAQEBAQEBAAAAAAAAAAABAgMEBQYHCAkKC//EALUQAAIBAwMCBAMFBQQEAAABfQECAwAEEQUSITFBBhNRYQcicRQygZGhCCNCscEVUtHwJDNicoIJChYXGBkaJSYnKCkqNDU2Nzg5OkNERUZHSElKU1RVVldYWVpjZGVmZ2hpanN0dXZ3eHl6g4SFhoeIiYqSk5SVlpeYmZqio6Slpqeoqaqys7S1tre4ubrCw8TFxsfIycrS09TV1tfY2drh4uPk5ebn6Onq8fLz9PX29/j5+v/EAB8BAAMBAQEBAQEBAQEAAAAAAAABAgMEBQYHCAkKC//EALURAAIBAgQEAwQHBQQEAAECdwABAgMRBAUhMQYSQVEHYXETIjKBCBRCkaGxwQkjM1LwFWJy0QoWJDThJfEXGBkaJicoKSo1Njc4OTpDREVGR0hJSlNUVVZXWFlaY2RlZmdoaWpzdHV2d3h5eoKDhIWGh4iJipKTlJWWl5iZmqKjpKWmp6ipqrKztLW2t7i5usLDxMXGx8jJytLT1NXW19jZ2uLj5OXm5+jp6vLz9PX29/j5+v/aAAwDAQACEQMRAD8A8zgHzAdM+vatbGLNvTYLeDUEYAlxz8sihQc9KzuzGTdj2azaea1V7i3Nu+PuMwJH1xxXoQk2tTjlEz5fFWjwmSP7UHkj6gKcH8elZyrwS0LhTcnY878TapPrepmeSSFYYhsg2jcpXJz9TnvXLKo5as6lFQ0RyWHKhimAVJB9aq5oy9pkhwWYAxhgD0HPUc9R0qJESR6rrXiW307QIoo3E1xPajaUYYUYxk/r+VbTqWikupzxhdnmV1qBFq0UPzCQDftOSM9RXNGOt2dChYxhPImMkjjgegrY0saen6dJPKDdyPHbRL8zAg7AfQU3GVm0LmjdJ9TovJ0m0eOO0eG9tWUK7SBtiHO75ehPP5etZavcqyuW2uYF8qd4YIGZvLRivK8HHTmoeug4JIrWvhi81bVUubWK2LkAsikMuAOTzjnrQ5pKyLUb6syJtB1LzzJ/ZsUyYx+5c4PvVqS7kG++jizu1tLRobqZRunWMllAHVSen5d6qM3yu+lzNx172K+rWm2dbGEi6Xyi7+UrKFG3PcdQP51K01ZSXYg1W3aW8j0u0u45hDH9sJUEbwqbgBnoSpI571KfUu1tDr/CkC2Gqv5s8It71NlnMSQZ9y7gY/wIzn6daiWqKirMqkC1vZbPU5Y7GeID5pHcCUeoKg5otpdC9S7q80Xg/QYLPTlbz7gsonbGRxlm+voOgpRvJ3ZTXKtCXwHYKumy6g53SXTmMZ5wqnnP1P8AIUpvWw4LS5g6VOJPHizCNR514ysNo5XlcflVNe6Jbmp47s7eKOzuII1iMDeUVRQoGfmUrjpgqf0pU3uhy2NbThY+M9Dgn1O0DywuyMwJU7h1II5wcg49aiV6crIa95an/9k=""")
```

Comment:
Simple way of reducing image size is to strip EXIF data: `$ exiftool -all= yourfile.jpg`. Here's the resulting base64 blob:
```suggestion
    demoIcon = base64.decode("""/9j/2wBDAAoHBwgHBgoICAgLCgoLDhgQDg0NDh0VFhEYIx8lJCIfIiEmKzcvJik0KSEiMEExNDk7Pj4+JS5ESUM8SDc9Pjv/2wBDAQoLCw4NDhwQEBw7KCIoOzs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozv/wAARCAAmACYDAREAAhEBAxEB/8QAHwAAAQUBAQEBAQEAAAAAAAAAAAECAwQFBgcICQoL/8QAtRAAAgEDAwIEAwUFBAQAAAF9AQIDAAQRBRIhMUEGE1FhByJxFDKBkaEII0KxwRVS0fAkM2JyggkKFhcYGRolJicoKSo0NTY3ODk6Q0RFRkdISUpTVFVWV1hZWmNkZWZnaGlqc3R1dnd4eXqDhIWGh4iJipKTlJWWl5iZmqKjpKWmp6ipqrKztLW2t7i5usLDxMXGx8jJytLT1NXW19jZ2uHi4+Tl5ufo6erx8vP09fb3+Pn6/8QAHwEAAwEBAQEBAQEBAQAAAAAAAAECAwQFBgcICQoL/8QAtREAAgECBAQDBAcFBAQAAQJ3AAECAxEEBSExBhJBUQdhcRMiMoEIFEKRobHBCSMzUvAVYnLRChYkNOEl8RcYGRomJygpKjU2Nzg5OkNERUZHSElKU1RVVldYWVpjZGVmZ2hpanN0dXZ3eHl6goOEhYaHiImKkpOUlZaXmJmaoqOkpaanqKmqsrO0tba3uLm6wsPExcbHyMnK0tPU1dbX2Nna4uPk5ebn6Onq8vP09fb3+Pn6/9oADAMBAAIRAxEAPwDzOAfMB0z69q1sYs29Ngt4NQRgCXHPyyKFBz0rO7MZN2PZrNp5rVXuLc274+4zAkfXHFehCTa1OOUTPl8VaPCZI/tQeSPqApwfx6VnKvBLQuFNydjzvxNqk+t6mZ5JIVhiGyDaNylcnP1Oe9csqjlqzqUVDRHJYcqGKYBUkH1qrmjL2mSHBZgDGGAPQc9Rz1HSokRJHquteJbfTtAiijcTXE9qNpRhhRjGT+v5VtOpaKS6nPGF2eZXWoEWrRQ/MJAN+05Iz1Fc0Y63Z0KFjGE8iYySOOB6CtjSxp6fp0k8oN3I8dtEvzMCDsB9BTcZWbQuaN0n1Oi8nSbR447R4b21ZQrtIG2Ic7vl6E8/l61lq9yrK5ba5gXyp3hggZm8tGK8rwcdOah66Dgkita+GLzVtVS5tYrYuQCyKQy4A5POOetDmkrItRvqzIm0HUvPMn9mxTJjH7lzg+9WpLuQb76OLO7W0tGhuplG6dYyWUAdVJ6fl3qozfK76XM3HXvYr6tabZ1sYSLpfKLv5SsoUbc9x1A/nUrTVlJdiDVbdpbyPS7S7jmEMf2wlQRvCpuAGehKkjnvUp9S7W0Ov8KQLYaq/mzwi3vU2WcxJBn3LuBj/AjOfp1qJaoqKsyqQLW9ls9TljsZ4gPmkdwJR6gqDmi2l0L1LurzReD9Bgs9OVvPuCyidsZHGWb6+g6ClG8ndlNcq0JfAdgq6bLqDndJdOYxnnCqec/U/wAhSm9bDgtLmDpU4k8eLMI1HnXjKw2jleVx+VU17oluanjuzt4o7O4gjWIwN5RVFCgZ+ZSuOmCp/SlTe6HLY1tOFj4z0OCfU7QPLC7IzAlTuHUgjnByDj1qJXpyshr3lqf/2Q==""")
```